### PR TITLE
revert: revert core genesis usage

### LINKED ISSUE #33: Do no use core genesis
Core genesis is not final, we should probably use the legacy genesis way.

### PR DESCRIPTION
**Pull request by julienrbrt**
Closes https://github.com/cosmosregistry/example/issues/33

### CODE REVIEW

File: module/module.go [hunk 1]
```diff
@@ -71,3 +72,42 @@ func (am AppModule) RegisterServices(cfg module.Configurator) {
...
+	if err := cdc.UnmarshalJSON(bz, &data); err != nil {
+		return fmt.Errorf("failed to unmarshal %s genesis state: %w", example.ModuleName, err)
+	}
+
+	return data.Validate()
+}
+
+// InitGenesis performs genesis initialization for the example module.
+// It returns no validator updates.
+func (am AppModule) InitGenesis(ctx sdk.Context, cdc codec.JSONCodec, data json.RawMessage) []abci.ValidatorUpdate {
```
Comment (julienrbrt) [inline module/module.go:93]: 😢 @tac0turtle, we need your PR :D

**Comment by julienrbrt** [comment]
@facundomedica, should we show a complete genesis example? By importing and exporting the counters per address?
Or should we keep showing an example about core genesis even though it will get improvements later on?

File: module/module.go (hunk 1 above)
Comment (tac0turtle) [reply module/module.go:93]: which pr is this?

Comment (julienrbrt) [reply module/module.go:93]: Discussed on Slack :)

**Pull request merged**